init.lua

```diff
@@ -74,6 +80,18 @@ function SkyRocket:new(options)
     windowCanvas = createResizeCanvas(),
     resizeModifiers = options.resizeModifiers or {'ctrl', 'shift'},
     targetWindow = nil,
+
+    fancyZoneModifier = options.fancyZoneModifier or {'shift'},
+    zones = options.zones or {
+      {w='0.50',h='0.50',x='0.0' ,y='0'},
+      {w='0.50',h='0.50',x='0.0',y='50'},
+      {w='0.50',h='0.50',x='0.50', y='0'},
+      {w='0.50',h='0.50',x='0.50',y='50'},
+    },
+    fancyPosition=hs.geometry.point(0,0),
+    showcanvas = false,
```

Comment:
Please be locally consistent, all other variables are `camelCased`

(rename to `showCanvas`)